filters/outline/src/OutlineFilter.ts

```diff
@@ -26,23 +26,26 @@ class OutlineFilter extends Filter
 
     private _thickness = 1;
     private _alpha = 1.0;
+    private _knockout = false;
 
     /**
      * @param {number} [thickness=1] - The tickness of the outline. Make it 2 times more for resolution 2
      * @param {number} [color=0x000000] - The color of the outline.
      * @param {number} [quality=0.1] - The quality of the outline from `0` to `1`, using a higher quality
      *        setting will result in slower performance and more accuracy.
      * @param {number} [alpha=1.0] - The alpha of the outline.
+     * @param {boolean} [knockout=false] - Only render outline, not the contents.
      */
-    constructor(thickness = 1, color = 0x000000, quality = 0.1, alpha = 1.0)
+    constructor(thickness = 1, color = 0x000000, quality = 0.1, alpha = 1.0, knockout = false)
```

Comment:
This constructor is getting a little unwieldy here. I'd recommend (maybe another PR) to use options object instead and deprecate the arguments style.

Reply:
Agreed, I think all filters with more than one setting should use an object.

Reply:
Perhaps fix all filters settings at the same time Color class is introduced. It will most likely require a major version bump.